src/Plugins/Pagination.js

```diff
@@ -99,6 +99,28 @@ class Pagination {
 	}
 
 	isFiltered(value) {
+		const hasInclude = "include" in this.data.pagination;
+		const hasExclude = "exclude" in this.data.pagination;
+		if (hasInclude && hasExclude) {
+			throw new Error("Pagination cannot have both `include` and `exclude` properties.");
+		}
+		if (hasInclude) {
+			let included = this.data.pagination.include;
+			if (Array.isArray(included)) {
+				return !included.includes(value);
+			}
+			return included !== value;
+		}
+		if (hasExclude) {
+			let excluded = this.data.pagination.exclude;
+			if (Array.isArray(excluded)) {
+				return excluded.includes(value);
+			}
+			return excluded === value;
+		}
+
+		// Let's keep this code for backwards compatibility to V2.
```

Comment:
And we can remove this bit now right?

Reply:
I'd be happy to do so, but I think this should be communicated, since it affects the public API of this plugin. Is there a way to log warnings if someone still uses filters?

Reply:
Maybe we add a warning when someone uses `filter` that says "The filter Pagination property has been deprecated in Eleventy v3. You can replace it with `exclude` for the same effect".

Reply:
Yes, exactly. My question is, if there is a better way than throwing an error or using console.warn for this.

Reply:
I don't see anywhere else where we are throwing an error or console.warn-ing for a deprecation. Seems like those just get tagged as deprecated with JSDoc but aren't removed or anything.

Reply:
Hey @zachleat, how would you want me to handle this here?
Maybe you even have an opinion on doing a breaking change.